src/main/java/com/bandwidth/sqs/consumer/Consumer.java

```diff
@@ -38,8 +36,8 @@
 
 public class Consumer {
```

Comment:
Shouldn't the consumer accept a generic type `<T>` that is the same one used on the `SqsQueue<T>` and `ConsumerHandler<T>`?
If not, the `SqsQueue` shouldn't demand a generic type but always use String instead.